demo/index.html

```diff
@@ -5,8 +5,8 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
     <title>&lt;google-youtube&gt; Demo</title>
-    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-    <link rel="import" href="demo-elements.html">
+    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
```

Comment:
for webcomponents.org purposes, please point this to the newest unpkg url of this file

Reply:
https://unpkg.com/@webcomponents/webcomponentsjs@2.2.1/webcomponents-bundle.js

Reply:
Done.
Thanks for providing the url.